graphql_compiler/tests/test_helpers.py

```diff
@@ -178,6 +178,8 @@ def get_schema():
             in_Animal_ImportantEvent: [Animal]
             in_Entity_Related: [Entity]
             out_Entity_Related: [Entity]
+            out_Event_RelatedEvent: [Event]
```

Comment:
These should be of union type, `EventOrBirthEvent`

Reply:
To my understanding, we want a structure such as the following:
subclass {
    out_superclass_edge_to_superclass_type{
        common property @output...
   }
}

Though I suppose this is a broader case than:
subclass {
    out_superclass_edge_to_superclass_or_subclass_type{
        common property @output...
   }
}

Reply:
I don't think `out_superclass_edge_to_superclass_or_subclass_type` is valid -- you can't recurse into a subclass type, because the starting point of the recursion isn't of that type (by definition).